src/odemis/acq/stream/_projection.py

```diff
@@ -526,6 +538,20 @@ def _updateImage(self):
         except Exception:
             logging.exception("Updating %s %s image", self.__class__.__name__, self.stream.name.value)
 
+    def projectAsRaw(self):
+        """ Project a raw image without converting to RGB
+
+        Handles tiles as well as regualr DataArray's
+        """
+        raw = self.stream.raw
+
+        if isinstance(raw[0], model.DataArrayShadow):
+            # DataArrayShadow => need to get each tile individually
+            self._raw, projected_tiles = self._getTilesFromSelectedArea()
```

Comment:
No, in such cases, get the raw tuples from `self._getTilesFromSelectedArea()`, merge them with mergeTiles(), and return that raw data.